traffic_ops/traffic_ops_golang/routes.go

```diff
@@ -71,55 +71,35 @@ func Routes(d ServerData) ([]Route, http.Handler, error) {
 		//About
 		{1.3, http.MethodGet, `about/?(\.json)?$`, about.Handler(), auth.PrivLevelReadOnly, Authenticated, nil},
 
-		// Proxied routes
```

Comment:
no need for these anymore because the http.MethodGet, `cdns/{id}$`, route was removed